src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
The class `RightTriangle` should not have an empty line at the beginning of its definition.